src/Operator/Operators.php

```diff
@@ -56,7 +56,10 @@ public static function gt(np_array $data, $arg)
         $result = [];
 
         foreach ($data as $index => $item)
-            $result[] = $item > $arg[$isSingle?0:$index];
+            if ($item instanceof np_array)
+                $result [] = self::gt($item, $arg);
+            else
+                $result[] = $item > $arg[$isSingle?0:$index];
```

Comment:
Variable `$isSingle?0:$index` is bad for understanding. Maybe you can use something else or add a good comment.